need import tokio_metrics and tokio_metrics_collector

### What type of bug is this?

Other

### What subsystems are affected?

Other

### Minimal reproduce step

just git clone main branch and make 

### What did you expect to see?

I just want to compile successfully.

### What did you see instead?

error[E0433]: failed to resolve: use of undeclared crate or module `tokio_metrics`
   --> src/common/runtime/src/runtime.rs:215:19
    |
215 |     let monitor = tokio_metrics::RuntimeMonitor::new(handle);
    |                   ^^^^^^^^^^^^^ use of undeclared crate or module `tokio_metrics`

error[E0433]: failed to resolve: use of undeclared crate or module `tokio_metrics_collector`
   --> src/common/runtime/src/runtime.rs:216:21
    |
216 |     let collector = tokio_metrics_collector::RuntimeCollector::new(monitor, name);
    |                     ^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `tokio_metrics_collector`

   Compiling async-lock v3.4.0
For more information about this error, try `rustc --explain E0433`.
error: could not compile `common-runtime` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
make: *** [build] Error 101

### What operating system did you use?

Mac OS X 15.1 ARM

### What version of GreptimeDB did you use?

0.11.1

### Relevant log output and stack trace

```bash
error[E0433]: failed to resolve: use of undeclared crate or module `tokio_metrics`
   --> src/common/runtime/src/runtime.rs:215:19
    |
215 |     let monitor = tokio_metrics::RuntimeMonitor::new(handle);
    |                   ^^^^^^^^^^^^^ use of undeclared crate or module `tokio_metrics`

error[E0433]: failed to resolve: use of undeclared crate or module `tokio_metrics_collector`
   --> src/common/runtime/src/runtime.rs:216:21
    |
216 |     let collector = tokio_metrics_collector::RuntimeCollector::new(monitor, name);
    |                     ^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `tokio_metrics_collector`

   Compiling async-lock v3.4.0
For more information about this error, try `rustc --explain E0433`.
error: could not compile `common-runtime` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
make: *** [build] Error 101
```

--- a/src/common/runtime/Cargo.toml
+++ b/src/common/runtime/Cargo.toml
@@ -39,3 +39,7 @@ tokio-util.workspace = true
 
 [dev-dependencies]
 tokio-test = "0.4"
+
+[target.'cfg(tokio_unstable)'.dependencies]
+tokio-metrics = { version = "0.3" }
+tokio-metrics-collector = { version = "0.2" }

